cfme/tests/automate/test_git_import.py

```diff
@@ -78,3 +78,69 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain, br
     view.modal.apply.click()
     assert view.provisioning_entry_point.value == ("/{}/Service/Provisioning/StateMachines/"
         "ServiceProvision_Template/CatalogItemInitialization".format(imported_domain.name))
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    ("url", "param_type", "param_value", "verify_ssl"),
```

Comment:
Optional: To avoid the lengthy in the marker of a test case, you could place this elsewhere in the file and write it as a `namedtuple`. See [here](https://github.com/ManageIQ/integration_tests/blob/86a6f2acd36508fecf57e12477be9d1bb16246ec/cfme/tests/control/test_basic.py#L94) for an example.